setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
I'd say I'm a rather big fan of comprehensions myself, but I don't think that using it like you do in this case is far from being efficient. Your comprehension itself generates a throw-away list of Nones, while the real work is done through appending to another list. So, yes, list comprehensions are pythonic and efficient, but only when used as inetended.